### ISSUE TITLE
Fine tuning only biases

### ISSUE THREAD

**Issue by Steven-Palayew** [amaiya/ktrain#433]
Hello, I was wondering what the easiest way to fine tune only the biases (not the weights) of a transformer would be with ktrain? Like what they propose here: https://arxiv.org/abs/2106.10199. 

Thanks in advance!

**Comment by amaiya** [comment]
Hello:  If you look at the code for the paper, they use the `requires_grad` flag in PyTorch to perform the relevant gradient deactivations.  You could do something similar in TensorFlow (e.g., `stop_gradient`).  But, you'll need to go to the TF documentation (and perhaps the **transformers** and **tensorflow** discussion forums) for more information.
